internal/docstore/dynamodocstore/create_tables.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Creates the DynamoDB tables needed for tests.
```

Comment:
Just want to make sure, this only needs to run when the table is somehow deleted right?

Reply:
Right, or if a contributor wants to run tests against their own Amazon account.